src/helpers.js

```diff
@@ -8,6 +8,8 @@
 import { GENESIS_BLOCK, DECIMAL_PLACES, DEFAULT_SERVER } from './constants';
 import path from 'path';
 
+const storage = require('./storage').default;
```

Comment:
I'd rather something like `import { defaultStorage as storage } from './storage';`.

Reply:
Sorry. I think the correct one would be `import defaultStorage from './storage'`;

Reply:
Done